x-pack/plugins/ml/common/util/job_utils.test.js

```diff
@@ -20,38 +20,38 @@ import {
   prefixDatafeedId,
   getSafeAggregationName,
   getLatestDataOrBucketTimestamp,
-} from '../job_utils';
+} from './job_utils';
 
 describe('ML - job utils', () => {
   describe('calculateDatafeedFrequencyDefaultSeconds', () => {
-    it('returns correct frequency for 119', () => {
+    test('returns correct frequency for 119', () => {
       const result = calculateDatafeedFrequencyDefaultSeconds(119);
       expect(result).to.be(60);
```

Comment:
Please remove `import expect from '@kbn/expect';` and use `expect` provided by jest. Don't forget to replace `.to.be` with `.toBe`.